e2e/scripts/start-verdaccio.sh

```diff
@@ -57,17 +57,31 @@ e2e_root_path=$PWD
 
 if [[ "$RELEASE" = true ]]; then
   # ****************************************************************************
-  # First, publish the monorepo.
+  # First, install emsdk.
   # ****************************************************************************
+  # tfjs-backend-wasm needs emsdk to build. emsdk install needs to be done
+  # before switch to local registry, otherwise some packages installation will
+  # fail.
+  # Todo(linazhao): Remove this once we have a custom docker with emsdk.
+  cd ..
+  git clone https://github.com/emscripten-core/emsdk.git
```

Comment:
how time consuming is this step? can the script ask for where the local installed version of emsdk is, install a new version otherwise?